packages/react/src/refs.tsx

```diff
@@ -163,22 +163,29 @@ function resolveRef(dataValues: DataValues, lookup: ReferencedDataState['ref']):
 export function Template({
   components,
   dataSource,
-  path = '',
+  storeKey = '',
   onEvent,
 }: {
-  path?: ReferencedDataState['ref']
+  storeKey?: string
```

Comment:
tbd: `storeKey` will be incorrect if there's another component referenced. If you ref component from another store, the storeKey should be of that store, otherwise it's not going to work. Will fix later.